go-fuzz-build/main.go

```diff
@@ -38,6 +38,7 @@ var (
 	flagRace      = flag.Bool("race", false, "enable race detector")
 	flagCPU       = flag.Bool("cpuprofile", false, "generate cpu profile in cpu.pprof")
 	flagLibFuzzer = flag.Bool("libfuzzer", false, "output static archive for use with libFuzzer")
+	flagBuildX    = flag.Bool("x", false, "print the commands")
```

Comment:
"print the commands" is a bit confusing because hopefully most builds succeed, so users won't see any difference and won't see the commands. So I think this "if build fails" part needs to be reflected in the descriptions (or -x should produce output always).

Reply:
You're right, have updated flag description.
Thanks!